docs/sources/panels/visualizations/_index.md

```diff
@@ -1,9 +1,9 @@
 +++
-title = "Visualization options"
+title = "Visualizations"
 weight = 400
 +++
 
-# Visualization options
+# Visualizations
```

Comment:
Would be nice to list all the visualizations here with an image that is clickable (link image) that take you to that visualization doc article. Something for another PR if anyone from docs team think this is a good idea. Thinking to use this index page as a quick overview of what visualizations exist and can look like?

Reply:
Sure, can do. Any reason to do this post v8? Feels like it makes sense to do for v8.

Reply:
I mean just the naming of the side menu navigation. the content for this index page can wait post v8

Reply:
With the change to the Visualizations so that it is separate from all the other options, one of the main things it now does is change which options are available in the options section. Previously, the size of the Visualizations section kept all those changes "below the fold." The thought is to call out everything as "XX options" reference topics and then make overarching tasks to step users through creating each type of panel.